src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -49,10 +57,15 @@ public function __construct($queryBuilder, $manager = null, $class = null)
         }
 
         if ($queryBuilder instanceof \Closure) {
+            trigger_error('Passing a QueryBuilder closure to '.__CLASS__.'::__construct() is deprecated since version 2.7 and will be removed in 3.0.', E_USER_DEPRECATED);
```

Comment:
OK, nevermind. the form itself still accepts closures. the resolution has only been moved to a different place